shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -171,7 +188,14 @@ - (BOOL)application:(UIApplication*)application
   if ([_lifeCycleDelegate application:application openURL:url options:options]) {
     return YES;
   }
-  return [self openURL:url];
+  if (![self isFlutterDeepLinkingEnabled]) {
```

Comment:
do we still ned this check if we go delegate to other selector in line 194?

Reply:
I just add this so the return value reflect the value of the flag.